frontend/hooks/useReward.ts

```diff
@@ -2,22 +2,4 @@ import { useContext } from 'react';
 
 import { RewardContext } from '@/context/RewardProvider';
 
-export const useReward = () => {
-  const {
-    availableRewardsForEpoch,
-    availableRewardsForEpochEth,
-    isEligibleForRewards,
-    accruedServiceStakingRewards,
-    isStakingRewardsDetailsLoading,
-    isStakingRewardsDetailsError,
-  } = useContext(RewardContext);
-
-  return {
-    availableRewardsForEpoch,
-    availableRewardsForEpochEth,
-    isEligibleForRewards,
-    accruedServiceStakingRewards,
-    isStakingRewardsDetailsLoading,
-    isStakingRewardsDetailsError,
-  };
-};
+export const useReward = () => useContext(RewardContext);
```

Comment:
```suggestion
export const useRewardContext = () => useContext(RewardContext);
```

and maybe can rename the file as well, wdyt?